app/services/workflow_service/disposable_capital_assessment.rb

```diff
@@ -0,0 +1,21 @@
+module WorkflowService
+  class DisposableCapitalAssessment < BaseWorkflowService
+    def call
+      calculate_liquid_capital
+      PropertyAssessment.new(@particulars).call
+      VehicleAssessment.new(@particulars).call
+      true
+    end
+
+    private
+
+    def calculate_liquid_capital
+      total_liquid_capital = 0.0
+
+      @particulars.request.applicant_capital.liquid_capital.bank_accounts.each do |acct|
```

Comment:
I wonder if in `BaseWorkflowService` we could create a method like 
```ruby
def request
  @particulars.request
end
```

It would make this line a bit shorter.

Or even
```ruby
def applicant_capital
  @particulars.request.applicant_capital
end
```

Reply:
Agreed.